sdks/java/io/rabbitmq/src/test/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIOTest.java

```diff
@@ -59,29 +65,46 @@
 public class RabbitMqIOTest implements Serializable {
   private static final Logger LOG = LoggerFactory.getLogger(RabbitMqIOTest.class);
 
+  private static final int ONE_MINUTE_MS = 60 * 1000;
+
   private static int port;
+  private static String defaultPort;
+
   @ClassRule public static TemporaryFolder temporaryFolder = new TemporaryFolder();
 
   @Rule public transient TestPipeline p = TestPipeline.create();
 
-  private static transient Broker broker;
+  private static transient SystemLauncher launcher;
 
   @BeforeClass
   public static void beforeClass() throws Exception {
     port = NetworkTestHelper.getAvailableLocalPort();
 
+    defaultPort = System.getProperty("qpid.amqp_port");
```

Comment:
If we utilize this more in the future we can create a `@Rule` for a qpid broker, but it didn't seem worth the refactor given the single test file for rabbit.

Reply:
I think it's simple enough as it is.